isaaclab.sh

```diff
@@ -183,10 +186,18 @@ extract_isaacsim_exe() {
 install_isaaclab_extension() {
     # retrieve the python executable
     python_exe=$(extract_python_exe)
+
+    # detect if we're in a uv environment
+    if [ -n "$VIRTUAL_ENV" ] && [ -f "$VIRTUAL_ENV/pyvenv.cfg" ] && grep -q "uv" "$VIRTUAL_ENV/pyvenv.cfg"; then
+        pip_cmd="uv pip install"
+    else
+        pip_cmd="${python_exe} -m pip install"
+    fi
```

Comment:
Nitpicking on bash - We don't have a linter for it, it seems.

```suggestion
    # detect if we're in a uv environment
    if [ -n "${VIRTUAL_ENV}" ] && [ -f "${VIRTUAL_ENV}/pyvenv.cfg" ] && grep -q "uv" "${VIRTUAL_ENV}/pyvenv.cfg"; then
        pip_cmd="uv pip install"
    else
        pip_cmd="${python_exe} -m pip install"
    fi
```